dwave/system/composites/linear_ancilla.py

```diff
@@ -0,0 +1,232 @@
+    :math:`J_{ij}` between the qubit and a neighboring qubit :math:`j` that has a 
+    large flux-bias offset.
+
+    Args:
+    child_sampler (:class:`dimod.Sampler`):
+        A dimod sampler, such as a :class:`~dwave.system.samplers.DWaveSampler()`, 
+        that has flux bias controls.
+
+    Examples:
+        This example submits a two-qubit problem consisting of linear biases with oposed sign 
+        and anti-ferromagnetic coupling. A D-Wave system solves it in the fast anneal 
+        protocol through the use of ancilla qubits.
+
+        >>> from dwave.system import DWaveSampler, EmbeddingComposite, LinearAncillaComposite
+        ...
+        >>> sampler = EmbeddingComposite(LinearAncillaComposite(DWaveSampler()))
+        >>> sampleset = sampler.sample_ising({0:1, 1:-1}, {(0, 1): 1}, fast_anneal=True)
+        >>> sampleset.first.energy
+        -3
+
+    """
+
+    def __init__(
+        self,
+        child_sampler: dimod.Sampler,
+    ):
+        self.children = [child_sampler]
+        self.parameters = child_sampler.parameters.copy()
+        self.properties = dict(child_properties=child_sampler.properties.copy())
+        self.nodelist = child_sampler.nodelist
+        self.edgelist = child_sampler.edgelist
+
+    def nodelist(self):
+        pass  # overwritten by init
+
+    def edgelist(self):
+        pass  # overwritten by init
+
+    children = None  # overwritten by init
+    """list [child_sampler]: List containing the structured sampler."""
+
+    parameters = None  # overwritten by init
+    """dict[str, list]: Parameters in the form of a dict.
+
+    For an instantiated composed sampler, keys are the keyword parameters
+    accepted by the child sampler and parameters added by the composite.
+    """
+
+    properties = None  # overwritten by init
+    """dict: Properties in the form of a dict.
+
+    Contains the properties of the child sampler.
+    """
+
+    @nonblocking_sample_method
+    def sample(
+        self,
+        bqm: dimod.BinaryQuadraticModel,
+        *,
+        h_tolerance: numbers.Number = 0,
+        default_flux_bias_range: tuple[float, float] = (-0.005, 0.005),
+        **parameters,
+    ):
+        """Sample from the provided binary quadratic model.
+
+        .. note::
+            This composite does not support the :ref:`param_autoscale` parameter; use the
+            :class:`~dwave.preprocessing.composites.ScaleComposite` for scaling.
+
+        Args:
+            bqm (:class:`~dimod.binary.BinaryQuadraticModel`):
+                Binary quadratic model to be sampled from.
+
+            h_tolerance (:class:`numbers.Number`):
+                Magnitude of the linear bias to be set directly on problem qubits; above this the bias
+                is emulated by the flux-bias offset to an ancilla qubit. Assumed to be positive. 
+                Defaults to zero.
+
+            default_flux_bias_range (:class:`typing.Sequence`):
```

Comment:
Update type to match the new function signature, or remove it from docs completely.